### ISSUE TITLE
Axis label moves (loses proper margin) on chart update

### ISSUE THREAD

**Issue by lucaslin** [novus/nvd3#946]
Upon toggling a legend/control or resizing, the x-axis label jumps up and collides with the tick labels (loses proper margin). 

See plunker:
http://plnkr.co/edit/EaQwQaz0vyhceJMwtwln?p=preview

**Comment by liquidpele** [comment]
Yup, this is probably another case where we need to use the _option thing internally so it doesn't get overridden on chart update...
